modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/java/task/Receipt.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.jenkins.results.parser.java.task;
+
+import com.liferay.jenkins.results.parser.JenkinsResultsParserUtil;
+
+import java.io.IOException;
+
+import java.util.ArrayList;
+
+/**
+ * @author Brittney Nguyen
+ */
+public class Receipt {
+
+	public static void main(String[] args) throws IOException {
+		String path = JenkinsResultsParserUtil.combine(
+			"/opt/dev/projects/github/liferay-portal/",
+			"modules/test/jenkins-results-parser/",
+			"src/test/resources/dependencies/input1.txt");
```

Comment:
To use a resource instead of a hard-coded path.

Move `src/test/resources/dependencies/input1.txt` 
to `src/main/resources/com/liferay/jenkins/results/parser/java/task/dependencies/input1.txt`

Change
`main(String[] args) throws IOException`

To
`main(String[] args) throws Exception`

Use the following code:
```
		Class<?> clazz = Receipt.class;

		URL resourceURL = clazz.getResource("dependencies/input1.txt");

		URI resourceURI = resourceURL.toURI();

		Path resourcePath = Paths.get(resourceURI);

		File resourceFile = resourcePath.toFile();

		String path = resourceFile.getPath();
```